tests/python/relay/test_tensorrt.py

```diff
@@ -523,43 +535,44 @@ def check_trt_used(graph):
         latency[model], res = test_model(model, i_data, input_shape, dtype, use_trt=True)
         _, ref_res = test_model(model, i_data, input_shape, dtype, use_trt=False, num_iteration=1)
         tvm.testing.assert_allclose(res.asnumpy(), ref_res.asnumpy(), rtol=1e-3, atol=1e-3)
-    
+
+    print('Overall runtime for models run with TRT+VM')
     for model in models:
         print(model, latency[model])
 
 def test_tensorrt_serialize():
     if should_skip():
         return
-    import mxnet
+
     from mxnet.gluon.model_zoo.vision import get_model
     block = get_model('resnet18_v1', pretrained=True)
+    i_data = np.random.uniform(0, 1, (1, 3, 224, 224)).astype('float32')
+
     mod, params = relay.frontend.from_mxnet(block, shape={'data': (1, 3, 224, 224)}, dtype='float32')
+
+    ## Never saved
+    mod = relay.tensorrt.EnableTrt(mod, params)
+    with relay.build_config(opt_level=3):
+        exec = relay.create_executor("vm", mod=mod, ctx=tvm.gpu(0), target="cuda")
+        ref_res = exec.evaluate()(*i_data, **params)
+
     # Compile
     mod = relay.tensorrt.EnableTrt(mod, params)
     with relay.build_config(opt_level=3):
-        graph, lib, params = relay.build(mod, "cuda")
+        vm_exec = relay.vm.compile(mod, target="cuda", params=params)
     # Serialize
-    with open('compiled.json', 'w') as f_graph_json:
-        f_graph_json.write(graph)
-    with open('compiled.params', 'wb') as f_params:
-        f_params.write(relay.save_param_dict(params))
-    lib.export_library('compiled.so')
-    # Deserialize
-    with open('compiled.json', 'r') as f_graph_json:
-        graph = f_graph_json.read()
-    with open('compiled.params', 'rb') as f_params:
-        params = bytearray(f_params.read())
-    lib = tvm.runtime.load_module("compiled.so")
-    # Run
-    mod = graph_runtime.create(graph, lib, ctx=tvm.gpu(0))
-    mod.load_params(params)
-    i_data = np.random.uniform(0, 1, (1, 3, 224, 224)).astype('float32')
-    for i in range(10):
-        mod.run(data=i_data)
+    code, lib = vm_exec.save()
```

Comment:
Previously, this was testing serialize to disk and then read back from that file. I don't think we are testing ser-deser here.